docs/installing/manual_install.md

```diff
@@ -728,6 +762,18 @@ production server**. Replace `www.example.com` with your domain name.
       -subj /CN=www.example.com
     chmod 640 /etc/ssl/certs/www.example.com.cert
 
+If you are running a thin cluster, you will need to edit the top section of
+your <code>nginx-ssl.conf.example</code> file so that it has a `server` line
+for each instance you are running (otherwise nginx will only know how to send
+requests to the first server process). If you started your cluster with 3 servers,
```

Comment:
I used 3 as it's used throughout the thin docs, I imagine - although I've not seen it explained anywhere - that it's to avoid tying up all the CPU resources running server processes but I'm not attached to it